crates/pera-node/src/lib.rs

```diff
@@ -1290,7 +1291,14 @@ impl PeraNode {
                     .await?;
             }
         }
-
+        epoch_store
+            .set_signature_mpc_manager(SignatureMPCManager::new(
+                Arc::new(consensus_adapter.clone()),
+                Arc::downgrade(&epoch_store),
+                config.max_active_mpc_instances,
+                epoch_store.committee().voting_rights.len(),
+            ))
+            .await?;
```

Comment:
Might want to give this code some space:
```suggestion

        epoch_store
            .set_signature_mpc_manager(SignatureMPCManager::new(
                Arc::new(consensus_adapter.clone()),
                Arc::downgrade(&epoch_store),
                config.max_active_mpc_instances,
                epoch_store.committee().voting_rights.len(),
            ))
            .await?;

```